src/Util/ColumnParser.php

```diff
@@ -64,7 +63,11 @@ public function parseFields($arguments)
             ];
 
             if ($length !== null) {
-                $fields[$field]['options']['limit'] = $length;
+                if (is_array($length)) {
+                    list($fields[$field]['options']['precision'], $fields[$field]['options']['scale']) = $length;
```

Comment:
@markstory This is what I am talking about in the previous review comment. Please check.
When a user puts `amount[4,2]` , it will take 4 as precision and 2 for scale. If the user puts `amount[4,2,22,30]` , everything after 2 gets discardeded because `list` will ignore the rest. This `options` will further go in and do the SQL generation stuff.